packages/web-components/src/components/va-button/va-button.css

```diff
@@ -0,0 +1,61 @@
+@import '../../mixins/buttons.css';
+
+:host {
+  display: inline-block;
+}
+
+.va-button {
```

Comment:
Since we're working in the shadow DOM can we just style the `button` element directly and not use the class?